frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -169,13 +180,13 @@ export default function ProgramOverview() {
                 <div className="flex flex-col gap-4 h-[250px]">
                     <StatsCard
                         title="Residents Enrolled"
-                        number="104"
+                        number={metrics.Enrollments.toString()}
                         label="residents"
                         tooltip="Placeholder data"
```

Comment:
@corypride Can you also remove the placeholder data tooltip and change the title to "Active Enrollments"?